### ISSUE TITLE
extra, large DBs

### ISSUE THREAD

**Issue by 2sn** [conradtchan/starfit#24]
A key feature that is missing and that may require some integration with the deployment, is the ability of have extra DBs - large DBs - that should

1) live on the server but not be part of the PyPI package

2) users can get / download independently, maybe with a script.

3) starfit (PyPI) to then use these

This would be critical - highest priority.  If you have any time to still look at that, it would be most appreciated.

I think this would entail to somehow have these DBs ignored or the PyPI page, have a script to download them as part of the package (and/or documentation how to do it) or how/where to put extra DBs, and, maybe, functionality in the code to use these extra files?

**Comment by dliptai** [comment]
This should be fairly straightforward.

Users can already use an environment variable to point their installation of starfit to a different data directory (implemented in https://github.com/conradtchan/starfit/pull/8).

We just need to give them instructions on downloading ALL the files with wget. e.g.
```
wget --show-progress -qr -np -nH -R 'robots.txt','*html*' https://starfit.org/data/
```

The existing `download-data.sh` script (and thus what is included in PyPI) only downloads files listed in `.hashlist`. So as long as you don't add big DBs to those, they won't be include in PyPI releases.

**Comment by 2sn** [comment]
@dliptai  maybe there should be function as part of the package to download these files?

**Comment by 2sn** [comment]
currently a directory can be set where the data lives, `STARFIT_DATA`, but maybe this should be to allow specifying an additional directory rather than a replacement?

**Comment by 2sn** [comment]
This should now be implemented, but my need testing.

**Comment by 2sn** [comment]
implemented by #47 and follow-ups
